spec/base_spec.rb

```diff
@@ -12,9 +12,15 @@
       end
 
       it "does not allow multiple enqueues" do
-        subject.do
-        subject.do
-        expect(subject).to have_queue_size_of(1)
+        threads = []
+        50.times do
+          threads << Thread.new do
+            subject.do
+            expect(subject).to have_queue_size_of(1)
```

Comment:
Just curious if you checked that without your changes this test fails.

Yesterday I tried to write something similar with threads and the test passed without any code changes, but maybe I didn't write the test correctly.

Reply:
Yep, if you checkout only the first commit with the spec, it fails. https://github.com/rainforestapp/queue_classic_plus/pull/56/commits/1484e2a2f72df634eed917a14f307d9379aa4e32